addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -175,15 +188,15 @@ private boolean authorize(boolean useHttps, String fqdn, String apiBaseUrl, Stri
             loginManager.setAppToken(configuration.appToken);
             loginManager.openSession();
             return true;
-        } catch (FreeboxException | InterruptedException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
```

Comment:
@kaikreuzer : as this is run by a scheduled job, if I don't catch "largely", the job will just crashed silently.